src/app/grid/grid-remote-filtering-sample/remote-filtering-sample.component.scss

```diff
@@ -31,4 +31,13 @@
 
 .badge-left {
     float: left;
+}
+
+:host {
+    ::ng-deep {
+        .igx-overlay__wrapper--modal {
+            background: none;
+            pointer-events: initial;
```

Comment:
You can still block all pointer events, just don't make the grid flicker (darkening it out). Also, this changes the sample, but the default in the grid still remains with the dark background. This should also be fixed in the product.